cif_ed.dic

```diff
@@ -0,0 +1,643 @@
+;
+         syn
+;
+         'synthetic' Absolute configuration has not been established by
+         anomalous-dispersion effects in diffraction measurements on the
+         crystal. The enantiomer has been assigned by reference to an
+         unchanging chiral centre in the synthetic procedure.
+;
+         unk
+;
+         'unknown' No firm chemical or a-d evidence for an assignment is
+         available. An arbitrary choice of enantiomer has been made.
+;
+         .
+;
+         Inapplicable.
+;
+
+save_
+
+save_diffrn.precession_angle
+
+    _definition.id                '_diffrn.precession_angle'
+    _alias.definition_id          '_diffrn_precession_angle'
+    _definition.update            2024-04-22
+    _description.text
+;
+    Angle at the sample of tilt from the normal direction of a radiation
+    beam moved in a precessional scan.
+;
+    _name.category_id             diffrn
+    _name.object_id               precession_angle
+    _type.purpose                 Measurand
+    _type.source                  Recorded
+    _type.container               Single
+    _type.contents                Real
+    _enumeration.range            0.0:
+    _units.code                   degrees
+
+save_
+
+save_diffrn.precession_angle_su
+
+    _definition.id                '_diffrn.precession_angle_su'
+    _alias.definition_id          '_diffrn_precession_angle_su'
+    _definition.update            2024-04-22
+    _description.text
+;
+    Standard uncertainty of _diffrn.precession_angle.
+;
+    _name.category_id             diffrn
+    _name.object_id               precession_angle_su
+    _name.linked_item_id          '_diffrn.precession_angle'
+    _type.purpose                 SU
+    _type.source                  Recorded
+    _type.container               Single
+    _type.contents                Real
+    _units.code                   degrees
+
+save_
+
+save_diffrn_measurement.method_precession
+
+    _definition.id                '_diffrn_measurement.method_precession'
+    _alias.definition_id          '_diffrn_measurement_method_precession'
+    _definition.update            2024-04-22
+    _description.text
+;
+    Yes or no flag indicating if the radiation beam illuminates the sample
+    at an angle that is rotated about a precession axis.
+;
+    _name.category_id             diffrn_measurement
+    _name.object_id               method_precession
+    _type.purpose                 Encode
+    _type.source                  Assigned
+    _type.container               Single
+    _type.contents                Code
+    loop_
+      _enumeration_set.state
+      _enumeration_set.detail
+         yes                      'Precession method is used.'
+         y                        'Abbreviation for "yes".'
+         no                       'Precession method is not used.'
+         n                        'Abbreviation for "no".'
+
+    _enumeration.default          no
+
+save_
+
+save_diffrn_measurement.integration
```

Comment:
Since the values are drawn from an enumerated list, maybe the item should be renamed to something like `_diffrn_measurement.integration_mode`,   `_diffrn_measurement.integration_type` or `_diffrn_measurement.integration_method`?

Reply:
Will include as an issue for community consideration.